content/en/examples/security/kind-with-cluster-level-baseline-pod-security.sh

```diff
@@ -68,3 +69,14 @@ spec:
         - containerPort: 80
 EOF
 kubectl apply -f -
+
+# Wait
+sleep 3
```

Comment:
nit: 3 seconds isn't long to process the output. Should this wait for a keypress instead? How is this script used?

Reply:
> How is this script used?

To be honest, I'm not sure we need to keep it.

Reply:
I changed the script to await user input. Another PR can drop the script if that's what we want.